examples/functions/timetable/Netex_52_planned_delay.xml

```diff
@@ -91,7 +91,7 @@
 						<AvailabilityCondition version="any" id="before_july">
 							<Description>regular run</Description>
 							<FromDate>2025-01-01T00:00:00</FromDate>
-							<ToDate>2025-06-30T23:59:00</ToDate>
+							<ToDate>2025-06-30T23:59:59</ToDate>
```

Comment:
The actual 23:59:59 is not required. That was already part of the documention. It may be T00:00:00